packages/common/src/store/player/slice.ts

```diff
@@ -22,6 +29,10 @@ export type PlayerState = {
   // E.g. replaying a track doesn't change uid or trackId, but counter changes.
   counter: number
 
+  // Keep 'playbackRate' in the store separately from the audio
+  // Playback rate of the audio element
```

Comment:
lower case :)

Reply:
lower case?

Reply:
yeah `audio playback` or was it `audioPlayback` and it got moved down?